http/multipart_file.go

```diff
@@ -13,7 +13,7 @@ import (
 // MultipartFile is multipart form file.
 type MultipartFile struct {
 	Name   string
-	File   io.Reader
+	File   io.ReadSeeker
```

Comment:
This field is used by `Client` too.

Most `io.Reader` implementations do not support seeking (including `bytes.Buffer` and `fs.File`), so it would force `Client` users to buffer all data somewhere before sending.